code/index.html

```diff
@@ -5,32 +5,73 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" />
-    <title>Technigo Project Cluedo</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap"
```

Comment:
This you can import into your CSS file instead. So you don't have to put it on every HTML page for larger projects. You do this by go to google fonts, choose import and then you get a code looking something similar to this @import ....
Place it at the top of the CSS page :)